torch/distributed/_tensor/api.py

```diff
@@ -447,14 +447,25 @@ def distribute_tensor(
             first rank of each dimension of the `device_mesh`.
 
     Returns:
-        A :class:`DTensor` object
+        A :class:`DTensor` or `XLAShardedTensor` object.
+
+    Note:
+        When initialize the DeviceMesh with the `xla` device_type, `distribute_tensor`
+        return `XLAShardedTensor` instead. see [link](https://github.com/pytorch/pytorch/issues/92909)
+        for more details.
```

Comment:
Given that this is currently experimental feature and we need more testing to ensure it works e2e, let's add a warning here too 
```
.. warning::
    The XLA integration is experimental and subject to change.
```

Reply:
Done